src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`